### PR TITLE
Release/v1.0.1

### PR DESCRIPTION
**Pull request by DanielAWood**
1.0.1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```
Comment (tbelmega) [inline CHANGELOG.md:8]: this should be 1.0.1, correct?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Digital Evidence Archive
+# Digital Evidence Archive :D
```
Comment (tbelmega) [inline README.md:1]: is that intentional?

**Review by chasechow7 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (DanielAWood) [reply README.md:1]: good catch, that shouldn't be there

File: CHANGELOG.md (hunk 1 above)
Comment (DanielAWood) [reply CHANGELOG.md]: fixed

File: README.md (hunk 2 above)
Comment (DanielAWood) [reply README.md]: fixed

**Review by tbelmega — APPROVED** [review]

**Pull request merged**